tests/integration/components/content-kit-editor/component-test.js

```diff
@@ -264,6 +264,27 @@ test('it adds a component in display mode to the content-kit editor', function(a
   assert.ok(this.$(`#demo-card-editor`).length, 'Card changed to edit mode');
 });
 
+test('it adds a card and removed an active blank section', function(assert) {
+  assert.expect(2);
+  this.registry.register('template:components/demo-card', hbs`
+    <div id="demo-card"><button id='edit-card' {{action editCard}}></button></div>
+   `);
+  this.set('cards', [
+    createComponentCard('demo-card')
+  ]);
+  this.set('mobiledoc', simpleMobileDoc(''));
+  this.render(hbs`
+    {{#content-kit-editor mobiledoc=mobiledoc cards=cards as |contentKit|}}
+      <button id='add-card' {{action contentKit.addCard 'demo-card'}}></button>
+    {{/content-kit-editor}}
+  `);
+
+  assert.equal(this.$('.content-kit-editor p').length, 1, 'blank section exists');
+  moveCursorTo(this, '.content-kit-editor p');
+  this.$('button#add-card').click();
+  assert.equal(this.$('.content-kit-editor p').length, 0, 'no blank section');
```

Comment:
perhaps assert that the card is added as the firstChild of `.content-kit-editor`?